app/components/agenda/agenda-detail/sidebar-item.hbs

```diff
@@ -20,76 +20,70 @@
+        </h4>
+        {{#unless @agendaitem.aboutToDelete}}
+          <div class="vlc-agenda-detail-sidebar__formally-ok-label auk-o-flex--align-end">
 
-              {{#if this.loadNewsletterVisibility.lastSuccessful}}
-                {{#if (not this.newsletterIsVisible)}}
-                  <span class="vlc-agenda-meta__recently-added">
-                    <Auk::Icon @name="not-visible" @size="small" />
-                    <EmberTooltip @side="top-start" @tooltipClass="auk-tooltip">
-                      <p>
-                        {{t "not-visible-in-newsletter"}}
-                      </p>
-                    </EmberTooltip>
-                  </span>
-                {{else}}
-                  {{!-- TODO: this icon is merely here as an invisible placeholder.
-                    This keeps element-height the same at all times in order to prevent "jumping" when data is being side-loaded.
-                    This should be tackled differently through modified design --}}
-                  {{!-- template-lint-disable no-inline-styles --}}
-                  <i
-                    class="ki ki-not-visible auk-u-mr-2 auk-u-opacity--1/2"
-                    style="visibility:hidden"
-                  ></i>
-                  {{!-- template-lint-enable no-inline-styles --}}
-                {{/if}}
-              {{else}}
-                {{!-- TODO: design loading state for newsletter visibility --}}
-                <i class="ki-circle-question auk-u-mr-2 auk-u-opacity--1/2">
-                  <EmberTooltip @side="top" @tooltipClass="auk-tooltip">
+            {{#if this.loadNewsletterVisibility.lastSuccessful}}
+              {{#unless this.newsletterIsVisible}}
+                <Auk::Pill @skin="blank" @layout="icon-only">
+                  <Auk::Icon @name="not-visible" />
+                  <EmberTooltip @side="top-end" @tooltipClass="tooltip-custom">
                     <p>
-                      {{t "loading"}}
+                      Op de website
                     </p>
                   </EmberTooltip>
-                </i>
-              {{/if}}
+                </Auk::Pill>
+              {{/unless}}
+            {{else}}
+              <Auk::Pill @skin="blank" @layout="icon-only">
+                <Auk::Icon @name="three-dots" />
+                <EmberTooltip @side="top-end" @tooltipClass="tooltip-custom">
+                  <p>
+                    {{t "loading"}}
+                  </p>
+                </EmberTooltip>
+              </Auk::Pill>
+            {{/if}}
 
-              {{#if this.subcase.confidential}}
-                <span>
-                  <Auk::Icon data-test-agenda-detail-sidebar-item-confidential={{true}} @name="lock-closed" @size="small" />
-                </span>
-              {{/if}}
-            </div>
+            {{#if this.subcase.confidential}}
+              <AuPill @skin="warning" @layout="icon-only" data-test-agenda-detail-sidebar-item-confidential={{true}}>
```

Comment:
@erikap Ik ga hier binnen (en in de container) nog een paar kleine aanpassingen doen:
- `vlc-agenda-detail-sidebar__formally-ok-label` (container) een andere (meer duidelijke) classname geven, de padding weg halen en witruimte tussen de pills voorzien.
- `AuPill` heeft geen optie tot `@layout` en binnen Appuniversum componenten lijkt het me nu ook een goed idee om dan ook de iconen binnen Appuniversum te gebruiken.